src/material/button/button.md

```diff
@@ -45,6 +45,20 @@ not.
 </button>
 ```
 
+### Interactive disabled buttons
+Native disabled `<button>` elements cannot receive focus and do not dispatch any events. This can
+be problematic in some cases, because it can prevent the app from the telling the user why the
+button is disabled. You can use the `disabledInteractive` input to style the button as disabled but
+allow for it to receive focus and dispatch events. The button will have `aria-disabled="true"` for
+assistive technology. The behavior can be configured globally through the `MAT_BUTTON_CONFIG`
+injection token.
+
+**Note:** using the `disabledInteractive` input can result in buttons that previously prevented
+actions to no longer do so, for example a submit button in a form. When using this input, you should
+guard against such cases in your component.
```

Comment:
Can we provide an actionable suggestion, e.g. setting `tabindex=0` (not sure if thats right)?

Reply:
I don't think that we can provide a general advice since it depends on what the button does.